internal/core/coordinate_transformer.hpp

```diff
@@ -112,15 +120,19 @@ class DoubleCoordinateTransformer : public CoordinateTransformer {
         return m_transformer_a.IJKPositionToAnnotation(ijkPosistionInCubeA);
     }
 
-    void to_cube_a_ijk_position(float* out_cube_a_position, float const* intersection_cube_position) const {
-        for (int index = 0; index < 3; ++index) {
-            out_cube_a_position[index] = intersection_cube_position[index] + this->m_intersection_zero_as_cube_a_index[index];
-        }
+    void to_cube_a_voxel_position(float* out_cube_a_position, float const* intersection_cube_position) const {
```

Comment:
Why does this take raw pointers and not a vector?

In fact, perhaps we should consider std::array for these?